frontend/src/scenes/onboarding/onboardingLogic.tsx

```diff
@@ -373,6 +393,11 @@ export const onboardingLogic = kea<onboardingLogicType>([
             actions.setAllOnboardingSteps([])
 
             if (step) {
+                // when loading specific steps, like plans, we need to make sure we have a billing response before we can continue
+                const stepsToWaitForBilling = [OnboardingStepKey.PLANS, OnboardingStepKey.PRODUCT_INTRO]
```

Comment:
Just to confirm my understanding

This will not block the loading of onboarding and the billing request should be loaded in the background for all steps except product intro and plans. a.k.at the only time the initial loading of onboarding will be blocked by billing is when the initial loaded step is plans product intro and plans.

But won't almost all users initial loaded step be product intro?